src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,33 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String result = "Report for period " + dateFrom + " - " + dateTo;
+
+        LocalDate localDateFrom = LocalDate.parse(dateFrom,
+                                    DateTimeFormatter.ofPattern("dd.MM.yyyy"));
+        LocalDate localDateTo = LocalDate.parse(dateTo,
+                                    DateTimeFormatter.ofPattern("dd.MM.yyyy"));
+
+        for (String username: names) {
+            int money = 0;
+            
```

Comment:
Don't start the body of a loop with an empty line. Remove the redundant empty line before the inner for-loop.